src/zope/app/apidoc/utilities.py

```diff
@@ -232,17 +234,19 @@ def getFunctionSignature(func, ignore_self=False):
     # By filling up the default tuple, we now have equal indeces for args and
     # default.
     if defaults is not None:
-        defaults = (placeholder,)*(len(args)-len(defaults)) + defaults
+        defaults = (placeholder,) * (len(args) - len(defaults)) + defaults
     else:
-        defaults = (placeholder,)*len(args)
+        defaults = (placeholder,) * len(args)
 
     str_args = []
 
     for name, default in zip(args, defaults):
         # Neglect self, since it is always there and not part of the signature.
         # This way the implementation and interface signatures should match.
         if name == 'self' and (isinstance(func, types.MethodType) or ignore_self):
-            continue
+            # NOTE: this is actually covered, and removing the condition will break
+            # tests. The coverage report doesn't show it, though, for some reason.
+            continue # pragma: no cover
```

Comment:
I think sometimes the coverage bytecode analyzer is defeated by CPython's peephole optimizer.